### PR TITLE
MenuItems empty area

### PR DESCRIPTION
**Pull request by kroQ**
It close menu when click on empty space

### CODE REVIEW

File: app/src/main/java/org/supla/android/NavigationActivity.java [hunk 1]
```diff
@@ -152,20 +153,23 @@ private RelativeLayout getMenuItemsLayout() {
...
             AddDeviceButton.setOnClickListener(this);
+            EmptySpaceButton.setOnClickListener(this);
 
             Typeface type = Typeface.createFromAsset(getAssets(),"fonts/OpenSans-Regular.ttf");
             SettingsButton.setTypeface(type);
             AboutButton.setTypeface(type);
             DonateButton.setTypeface(type);
             HelpButton.setTypeface(type);
             AddDeviceButton.setTypeface(type);
+            EmptySpaceButton.setTypeface(type);
```
Comment (przemyslawzygmunt) [inline app/src/main/java/org/supla/android/NavigationActivity.java]: setTypeface is redundant

File: app/src/main/java/org/supla/android/NavigationActivity.java [hunk 2]
```diff
@@ -152,20 +153,23 @@ private RelativeLayout getMenuItemsLayout() {
...
             AddDeviceButton = MenuItemsLayout.findViewById(R.id.btn_add);
+            EmptySpaceButton = MenuItemsLayout.findViewById(R.id.btn_empty_space);
 
             SettingsButton.setOnClickListener(this);
             AboutButton.setOnClickListener(this);
             DonateButton.setOnClickListener(this);
             HelpButton.setOnClickListener(this);
             HomepageButton.setOnClickListener(this);
             AddDeviceButton.setOnClickListener(this);
+            EmptySpaceButton.setOnClickListener(this);
```
Comment (przemyslawzygmunt) [inline app/src/main/java/org/supla/android/NavigationActivity.java]: Is this really needed if onClick method do nothing for this button ?

File: app/src/main/java/org/supla/android/NavigationActivity.java [hunk 3]
```diff
@@ -176,6 +180,7 @@ private RelativeLayout getMenuItemsLayout() {
             HelpButton.setTransformationMethod(null);
             HomepageButton.setTransformationMethod(null);
             AddDeviceButton.setTransformationMethod(null);
+            EmptySpaceButton.setTransformationMethod(null);
```
Comment (przemyslawzygmunt) [inline app/src/main/java/org/supla/android/NavigationActivity.java]: Is this really needed for this button ?

File: app/src/main/java/org/supla/android/NavigationActivity.java [hunk 4]
```diff
@@ -459,6 +464,8 @@ public void onClick(View v) {
 
             openHomepage();
 
+        } else if ( v == EmptySpaceButton ) {
```
Comment (przemyslawzygmunt) [inline app/src/main/java/org/supla/android/NavigationActivity.java]: Is this really needed if there is no code inside ?

File: app/src/main/java/org/supla/android/NavigationActivity.java (hunk 2 above)
Comment (kroQ) [reply app/src/main/java/org/supla/android/NavigationActivity.java]: Yes, otherwise @Override onClick will not recognize that this button was clicked.

Comment (przemyslawzygmunt) [reply app/src/main/java/org/supla/android/NavigationActivity.java]: ok, I was fooled by the empty code in onClick. You probably meant about https://github.com/SUPLA/supla-android/blob/171d9e5ec3514214607e553e62902dd54078c6da/app/src/main/java/org/supla/android/NavigationActivity.java#L408

Comment (kroQ) [reply app/src/main/java/org/supla/android/NavigationActivity.java]: Yep.

File: app/src/main/java/org/supla/android/NavigationActivity.java (hunk 4 above)
Comment (kroQ) [reply app/src/main/java/org/supla/android/NavigationActivity.java]: Right, it is unneeded. I deleted it.

File: app/src/main/java/org/supla/android/NavigationActivity.java (hunk 3 above)
Comment (kroQ) [reply app/src/main/java/org/supla/android/NavigationActivity.java]: Right, it is unneeded. I deleted it.

File: app/src/main/java/org/supla/android/NavigationActivity.java (hunk 1 above)
Comment (kroQ) [reply app/src/main/java/org/supla/android/NavigationActivity.java]: Right, it is unneeded. I deleted it.

**Pull request merged**